fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVRewriteUtil.java

```diff
@@ -48,15 +48,6 @@ public static Collection<Partition> getMTMVCanRewritePartitions(MTMV mtmv, Conne
             long currentTimeMills) {
         List<Partition> res = Lists.newArrayList();
         Collection<Partition> allPartitions = mtmv.getPartitions();
-        // check session variable if enable rewrite
-        if (!ctx.getSessionVariable().isEnableMaterializedViewRewrite()) {
```

Comment:
why delete？

Reply:
Because we would support the mv rewrite in dml, it is controlled by `enable_dml_materialized_view_rewrite`.
So delete the logic, and this session variable is controlled by high level caller